packages/cli/generators/controller/index.js

```diff
@@ -46,6 +46,7 @@ module.exports = class ControllerGenerator extends ArtifactGenerator {
   }
```

Comment:
at the start of `scaffold()` after `super.scaffold()`, we need to call `if (this._shouldSkip()) return` again because the code beneath `super.scaffold()` will be run if it's not there. Here's where it unexpectedly errored out; there should probably be a test to make sure this doesn't happen.

Reply:
+1.